packages/vant/src/date-picker/DatePicker.tsx

```diff
@@ -121,6 +134,14 @@ export default defineComponent({
       return genOptions(minDate, maxDate, 'day', props.formatter, props.filter);
     };
 
+    const confirm = () => {
+      return pickerRef.value?.confirm();
+    };
+
+    const getSelectedDate = (): Array<string> => {
```

Comment:
The type of **getSelectedDate** can be inferred by ts, maybe no need to declare the return type.